src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -415,4 +418,20 @@ default boolean damage(double damage, DamageSource damageSource) {
         return getWorld().getEntities(predicate::test);
     }
 
+    /**
+     * Gets the current display name for this {@link Entity}.
+     *
+     * @return The display name
+     */
+    Text getDisplayName();
```

Comment:
@gabizou No, it isn't. This is correct, since `Text` is what we use to represent `IChatComponent` as well.
`getDisplayName` returns different values based on what `Keys.DISPLAY_NAME` is set.